Makefile

```diff
@@ -36,6 +36,7 @@ start:
 
 build:
 	@ printf "\n[${GREEN} Generating environment variables... ${NO_COLOR}]\n"
+	pip install git+https://github.com/ONSdigital/sdx-common#egg=sdx-common
```

Comment:
Alternative to this would be to add sdx-common to `REPOS` list and install it like this:
`pip install ${SDX_HOME}/sdx-common`